File: help/pt_pt/exercicios_da_ficha_11/Ficha11_009/Ficha11_009.cs
// Programador           -

using System.IO;
using System;
public class Ficha11_009 {

    //Main Function
    static void Main() {
        Console.Write("Altura : " );
        int altura = int.Parse(Console.ReadLine());
        triangulo(altura ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void triangulo( int altura){
        for( int i = 1; i < altura + 1; i = i + 1 ) {
            linha(i ) ;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:10

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_009/Ficha11_009.js
// Programador           -

//Main Function
function main() {
    document.write("Altura : " );// print text
    altura = parseInt(prompt("Altura : " ,"0") );
    
    document.write(altura + "\n");// print input 
    triangulo(altura ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linha(n) {
    for( var i = 0; i < n; i = i + 1 ) {
        document.write("*" );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function triangulo(altura) {
    for( var i = 1; i < altura + 1; i = i + 1 ) {
        linha(i ) ;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:10
